TPL/force-app/main/default/flexipages/Case_Individual_Lightning_Record_Page.flexipage-meta.xml

```diff
@@ -108,28 +108,8 @@
     <flexiPageRegions>
         <itemInstances>
             <componentInstance>
-                <componentInstanceProperties>
-                    <name>dir</name>
-                    <value>ltr</value>
-                </componentInstanceProperties>
-                <componentInstanceProperties>
-                    <name>inline</name>
-                    <value>false</value>
-                </componentInstanceProperties>
-                <componentInstanceProperties>
-                    <name>inlineLabel</name>
-                    <value>Launch docGenerationSample/fndSingleDocxLwc</value>
-                </componentInstanceProperties>
-                <componentInstanceProperties>
-                    <name>inlineVariant</name>
-                    <value>brand</value>
-                </componentInstanceProperties>
-                <componentInstanceProperties>
-                    <name>layout</name>
-                    <value>lightning</value>
-                </componentInstanceProperties>
-                <componentName>docGenerationSampleFndSingleDocxLwcEnglish</componentName>
-                <identifier>c_docGenerationSampleFndSingleDocxLwcEnglish</identifier>
```

Comment:
For the flyout process to be implemented, the process requires a flex card flyout which then calls the omniscript in a modal.